.circleci/config.yml

```diff
@@ -512,7 +512,7 @@ jobs:
           name: deploy
           command: |
             if [ $AZURE_STORAGE_SAS_TOKEN ]; then
-              azcopy copy "/tmp/verdaccio_dist/*/verdaccio/*" "https://reactspectrum.blob.core.windows.net/reactspectrum/$CIRCLE_SHA1/verdaccio${AZURE_STORAGE_SAS_TOKEN}" --recursive
+              azcopy copy "/tmp/verdaccio_dist/*/verdaccio/*" "https://reactspectrum.blob.core.windows.net/reactspectrum/${CIRCLE_SHA1}/verdaccio${AZURE_STORAGE_SAS_TOKEN}" --recursive
```

Comment:
Should `${AZURE_STORAGE_SAS_TOKEN}` be appended to the end of the url here? Wouldn't this mean the generated url would have said (secret?) token appended to the end?

Reply:
Pretty sure this is a standard API for moving files into blob storage. The URL won't get exposed publicly (unless we logged it out somewhere explicitly). I'm pretty sure the azure-cli was doing this under the hood, you just need to do it yourself with AzCopy's API:  https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-blobs-upload?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json#get-started

Reply:
gotcha thanks for the link! Not sure why the comment didn't seem to get made on the commit though, do you have a link for the generated test apps? I tried replacing a working build link commit's hash but no luck

Reply:
Not sure why comments aren't showing up, I'm having the same issue over at https://github.com/adobe/react-spectrum/pull/7027

So I've been replacing the hash in an old link like you mentioned. The deploy step logs show the files were successfully moved, so not sure where they are ending up.

Reply:
Ah, the comment on the commit isn't showing up because https://github.com/adobe/react-spectrum/blob/73414999fcc6afab5c4bc8ec4740fe3dc1eb8502/.circleci/comment.js#L14 needs to be true as well as https://github.com/adobe/react-spectrum/blob/73414999fcc6afab5c4bc8ec4740fe3dc1eb8502/.circleci/comment.js#L44. That being said, that will just comment the links with the commit hash which is what we've been trying to access manual with no luck :/.

Reply:
I think the issue may be with the wildcard in the path: https://stackoverflow.com/questions/44395658/azcopy-wildcards-in-middle-of-pattern